src/main/java/messenger/ImageMessengerNotification.java

```diff
@@ -0,0 +1,63 @@
+package messenger;
+
+import notification.Notification;
+
+import java.time.LocalDateTime;
+
+/**
+ * Created by annterina on 11.11.17.
+ */
+public class ImageMessengerNotification implements Notification {
+
+    private String senderId;
+    private String receiverId;
+    private String senderName;
+    private String receiverName;
+    private LocalDateTime sendDate;
+    private String imageName;
+    private long imageSize;
+    private byte[] image;
+
+    public ImageMessengerNotification(String senderId, String receiverId, String senderName, String receiverName, LocalDateTime sendDate, String imageName, long imageSize, byte[] image) {
```

Comment:
This should be package-private since you have a builder. When adding builder you want others to use only builder, so you want to limit access to this constructor by changing it from public to package-private (no keyword)

Reply:
Ok, that's right!

Reply:
sooo this is still public, right? or I don't know how to see current version of this file xD